db_service/dao_generator.go

```diff
@@ -100,6 +100,22 @@ func main() {
 				"RequestDetails":    `{"some":["json","blob","here"]}`,
 			},
 		},
+		{
+			Type:            "PlanDetailsV1",
```

Comment:
Does the `v1` here mean the same as `legacy`? Later in `three_to_four.go` you use the word `legacy` in functions which call this

Reply:
The v1 here refers to the specific version of the struct that's going to be deserialized. The struct is officially deprecated and unused except for compatibility purposes, which is why I'm referencing the V1 instance explicitly rather than using the proxy like the others. When v5 rolls around we should be able to get rid of it entirely with the compatibility layer.